test/test_dbcore.py

```diff
@@ -351,7 +351,7 @@ def pqp(self, part):
             part,
             {'year': dbcore.query.NumericQuery},
             {':': dbcore.query.RegexpQuery},
-        )
+        )[:-1]  # remove the negate flag
```

Comment:
Actually, this change was introduced because the `pqp(self, part)` method is used by that particular TestCase for making assertions on the tuple returned by `parse_query_part()`. Those tests were failing because it was expecting the "old" version of the tuple, with three elements.

It might be cleaner to modify all the tests on `QueryParseTest` instead, by appending `False` to the `r` tuples used for comparison?